tests/app_engine/main.py

```diff
@@ -43,16 +52,22 @@ def teardown():
     namespace = payload_dict['namespace']
     project_id = payload_dict["project_id"]
     print(f"starting teardown process for {namespace}")
+    # delete client and bootstrap deployments
+    try:
+        remove_client_deployments(
+            payload_dict["project_id"], payload_dict["cluster_name"], payload_dict["node_pool_zone"], namespace)
+    except Exception as e:
+        print(f"got an exception while trying to delete deployments under {namespace}:\n{e}")
```

Comment:
If the deployment removal failed, we really want to continue? Is there a chance that removing the node pool will succeed?

Reply:
we didn't remove deployments so far so, it won't affect the node pool deletion